### ISSUE TITLE
App icon won't change

### ISSUE THREAD

**Issue by LisaJo48** [OfficeDev/microsoft-teams-apps-champion-management#134]
I have replaced the star icon with one of our own with properties of 32x32 pixels and .jpg. The icon in the app remains the star. I even removed it from the CMP Logo library, deleted the browser history, cleared all cache but it still shows the star. What am I doing wrong?

![image](https://user-images.githubusercontent.com/46016140/197253454-992c98e7-7576-4b54-b39f-235bc66099e9.png)
![image](https://user-images.githubusercontent.com/46016140/197253558-fb99a216-7653-4a00-bb1b-a3561d4bd357.png)

**Comment by LisaJo48** [comment]
Update. Now one star icon is missing but the other is not and my logo is not in either spot. What I need, at the least, is that the logo right above the Welcome banner is updated to ours.
![image](https://user-images.githubusercontent.com/46016140/197634798-6dd8c13d-4344-4591-9c9f-6e4f5c859288.png)

**Comment by v-prigunasek** [comment]
Hi @LisaJo48 ,

Can you please rename the image to AppLogo.jpg in the CMP Logo library? Please refer the specifications in the document -
https://github.com/OfficeDev/microsoft-teams-apps-champion-management/wiki/Solution-Overview#manage-app-logo

After renaming the image, please clear the cache and try again. Thanks!

**Comment by LisaJo48** [comment]
NEVER caught that as I read the instructions over and over. Never crossed my mind to change the original's name and reuse it for the new one. So Sorry. Thanks!

So, is it correct that only one start icon has changed but not the others?
![image](https://user-images.githubusercontent.com/46016140/198135013-afdef1d6-78c4-444c-a8af-4944b1b07c9f.png)

**Comment by v-saikirang** [comment]
Hi @LisaJo48,

The Icon you pointed is coming from Teams App Store. So it cannot be changed from within the app. You will have to perform additional steps to change the icon in Teams store too:

1) Delete the app from "Manage Apps" section in "Teams Admin center". Note that this step will not make any changes to your Share Point site.
2) Download the below 3 files from the [Teams Folder](https://github.com/OfficeDev/microsoft-teams-apps-champion-management/tree/main/teams) in GitHub Repo.
![image](https://user-images.githubusercontent.com/87538202/198204002-f3240343-dc4a-4ff2-acf6-2b4e13b97881.png)
3) Replace the 2 image files with your own icons as per guidelines specified [here](https://learn.microsoft.com/en-us/microsoftteams/platform/concepts/build-and-test/apps-package).
4) Create a new ZIP file with the name "Champion Management Platform" with 3 files including 2 icons and manifest file. Note that the final zip package should contain only these 3 files without any nested folders.
5) Upload the publish the zip file from "Manage Apps" section in "Teams Admin Center"
6) Close browsers and reopen new browser after 5-10 mins to clear out cache.

Please let us know if you need more information.
Thanks.

**Comment by LisaJo48** [comment]
Thank you but, before engaging on the next change, I have an interesting observation. In the TEAMS via browser (Edge), my newest icon is showing (a yellow logo with a black bar underneath it) where I would expect it to show but, the Star icon is not showing in the other two locations mentioned above. However, in TEAMS on desktop, it shows the previous icon I'd tested, instead of the newest one, and the Start icon does show in the other two locations.

![image](https://user-images.githubusercontent.com/46016140/198718601-ce0d5777-b2c1-408b-a3c1-c035af2a9a86.png)

I have cleared the browser cache (images and files), refreshed several times, closed/reopened, and rebooted my computer. 
![image](https://user-images.githubusercontent.com/46016140/198720391-267379a0-349d-467e-9dc9-876bc5c21fa4.png)

So far, no change. 

However, doing the same steps in the Chrome browser, works as expected:
![image](https://user-images.githubusercontent.com/46016140/198720184-25add68b-15f8-4ad8-b975-248ad2ed88dc.png)

Firefox is also correct showing the Star icon in the two expected locations. I cannot test for Safari.

FYI: the Star icon was also in the Edge browser, as expected, until about 3am Wednesday. After that, it has looked like this. I made no changes other than upload a new image file and rename it, per instructions, to AppLogo.jpg
